### DISCUSSION TITLE
One-To-Any Relationship

### DISCUSSION THREAD

**Discussion by TheAzack9** [directus/directus#4749] [Feature Requests]
This would be a very handy tool to connect different collections to some selection. The idea stems from the new Tree-View where you might want to make a tree where each node is connected to some collection. I can see this being quite useful in any setting where you have some split content and want to just select one item. 

A user story:
As an example you want to make a navigation tree (So that you can create a navbar on your website) and you have different types of content on your webpage. One type is e.g  a generic page, while another is a news story. The way this was structured was to create two different collections "Page" and "News". You then create another collection called "Navigation Tree" which is supposed to mirror your nav bar. Each node in the nav tree then needs to be able to connect to either a singular page or a singular news story to be correctly mapped. A One-To-Any relationship would sove this! 

I am sorry if the explaination was horrible, it is not my strongest suit 😅

**Comment by robclancy** [comment]
Yep, I want to make links like in prismic using this.

**Reply by rijkvanzanten** [reply]
@robclancy Do you have an example of this "Link" style from Prismic? Maybe a reference to their docs, or a real-life use case example?

**Reply by robclancy** [reply]
![image](https://user-images.githubusercontent.com/857111/173590672-bb2c272d-6db7-414d-a73d-55bb50135cff.png)

https://prismic.io/docs/core-concepts/link-content-relationship

**Reply by rijkvanzanten** [reply]
So it's effectively a many to any limited to one item from the looks of it 👍🏻

**Reply by robclancy** [reply]
Yep, which is what I read this topic as but it's backwards lol

**Reply by robclancy** [reply]
Got here from this one https://github.com/directus/directus/discussions/5083. Didn't read that it said one to any and not many to single any.

**Reply by rijkvanzanten** [reply]
haha yeah the naming gets confusing pretty quickly. At least now I know which direction you're talking about specifically though 🙂 

I believe what you're referring to is what we internally refer to as the "any-to-one" which is effectively what the junction table uses in a many to any:

many-to-many = one-to-many + many-to-one
many-to-any = one-to-many + any-to-one

The relationship type already exists under the hood, but there's currently no GUI to configure / interact with it yet 👍🏻

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
